src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24878.cs

```diff
@@ -14,8 +15,11 @@ public Issue24878(TestDevice device) : base(device) { }
 	[Category(UITestCategories.ToolbarItem)]
 	public void ToolbarItemsShouldBeVisible()
 	{
+		App.WaitForElement("button");
+		App.Click("button");
 		App.WaitForElement("label");
 
 		VerifyScreenshot();
```

Comment:
Is it possible to add a screenshot before and after?

```suggestion
		App.WaitForElement("button");
		VerifyScreenshot("ToolbarItemsShouldBeVisible_Before");

		App.Click("button");

		App.WaitForElement("label");
		VerifyScreenshot("ToolbarItemsShouldBeVisible_After");
```

This way we can see what it would look like before the change and ensure that it did actually chnage.